src/full/cache.clj

```diff
@@ -98,87 +100,93 @@
 
 (defn rget
   [k & {:keys [throw?]}]
-  (when @client
+  (let [kp (prefixkey k)]
```

Comment:
you have 2 nested `let`s here, I would have moved prefix key to the inner let as the first binding

Reply:
ah I didnt even notice. I can fix

Reply:
fixed here and a few other places